changelogs/fragments/49235-docker_swarm_service-user-default.yaml

```diff
@@ -1,5 +1,5 @@
 ---
 bugfixes:
-  - 'fix: fails because of default "user: root" (https://github.com/ansible/ansible/issues/49199)'
+  - 'docker_swarm_service: fails because of default "user: root" (https://github.com/ansible/ansible/issues/49199)'
 minor_changes:
-  - 'docker_swarm_service: use docker defaults for the C(user) parameter if it is set to None'
+  - 'docker_swarm_service: use docker defaults for the C(user) parameter if it is set to C(null)'
```

Comment:
Ah, sorry for not noticing it earlier, but here you have to use ReST syntax, i.e.
```
   - 'docker_swarm_service: use docker defaults for the ``user`` parameter if it is set to ``null``' 
```

Reply:
No problem, thank you :)